packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -1,9 +1,5 @@
 const WORDPRESS_NAMESPACE = '@wordpress/';
-const BUNDLED_PACKAGES = [
-	'@wordpress/icons',
-	'@wordpress/interface',
-	'@wordpress/style-engine',
-];
+const BUNDLED_PACKAGES = [ '@wordpress/icons', '@wordpress/interface' ];
```

Comment:
I have just realized that is probably a breaking change from the perspective of the webpack plugin when developing for WordPress that still doesn't have `wp.styleEngine` exposed. I didn't think about it before. It will work correctly only when the Gutenberg plugin is enabled until WordPress 6.1 is out.

Reply:
Oh, good call. Thanks for spotting. 

Should I revert it for now?

Reply:
Actually, it might not be an issue at all if people don't reference the `@wordpress/style-engine` in their code. How likely it is, that plugin would use this new package? Maybe, it's enough to leave a note in both the package and the webpack plugin that it will work with WordPress 6.1 and above.

Reply:
`DependencyExtractionWebpackPlugin`, yes this package. It's also the context where I left the message so I assumed we talk about it :)

Maybe, we can simply add a note in the CHANGELOG file:

```
### New Features

- Include `@wordpress/style-engine` on the list of external dependencies to allow using `wp.styleEngine` global with WordPress 6.1 and beyond ([#43840](https://github.com/WordPress/gutenberg/pull/43840)).
```

Reply:
That's awesome, thank you! 🙇 

>  It's also the context where I left the message so I assumed we talk about it :)

Ha, no worries. I thought so, but I like to double check because I'm likely to add a comment in the `.nvmrc` file or somewhere else silly if I don't make sure 🙃